packages/pigeon/pigeons/core_tests.dart

```diff
@@ -12,6 +12,20 @@ enum AnEnum {
   fourHundredTwentyTwo,
 }
 
+enum AnotherEnum {
```

Comment:
I think it's worth commenting that this is because enums often require extra logic in generators, and having two enums ensures that the logic can be replicated without collisions.